README.md

```diff
@@ -16,9 +16,7 @@ Here is a summary of features enabled by this cost model:
 ## Requirements
 
 - Kubernetes version 1.8 or higher
-- kube-state-metrics
-- Node exporter
```

Comment:
This is true in the context of exporter-only, right? But not regular installs?

Reply:
So node-exporter is 100% not required in any mode.

I believe that ksm is "optional" but it does give you extra metrics, e.g. PVs. So maybe we drop node-exporter and caveat ksm?

Reply:
cc @AjayTripathy just in case he has a different perspective

Reply:
> So maybe we drop node-exporter and caveat ksm?

I agree with this idea.